students/shibin_mathew/lesson3/mailroom.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python3
+import sys
+
+
+def optionOne():
+    # Function to Send a Thank You Email
+    quit = False
+    while not quit:
+        print("Please input your full name")
+        name = input()
+        if name == "list":
+            print("list of names")
+            for names in donors:
+                print(names['name'])
+            print("Please select from the list or add a new name")
+            name = input()
+            if name != 'quit':
+                print("Please enter a donation amount")
+                donation = input()
+                if donation != 'quit':
+                    for names in donors:
+                        if names['name'] == name:
+                            names['donation'].append(float(donation))
+                    print(donors)  # Debug
```

Comment:
this currently has a bug that if you never issue `list` command and try to add to existing donor it just duplicates the donor instead of appending.